### PR TITLE
Add tx options param to prepare_claim_outputs()

### PR DESCRIPTION
**Pull request by Thoralf-M**
# Description of change

Add tx options param to prepare_claim_outputs() so one can set the address to which the outputs will be claimed

## Type of change

- Enhancement (a non-breaking change which adds functionality)

### CODE REVIEW

File: bindings/core/src/method/wallet.rs [hunk 1]
```diff
@@ -226,7 +226,11 @@ pub enum WalletMethod {
     /// Claim outputs.
     /// Expected response: [`PreparedTransaction`](crate::Response::PreparedTransaction)
     #[serde(rename_all = "camelCase")]
-    PrepareClaimOutputs { output_ids_to_claim: Vec<OutputId> },
+    PrepareClaimOutputs {
+        output_ids_to_claim: Vec<OutputId>,
+        #[serde(default)]
```
Comment (DaughterOfMars) [inline bindings/core/src/method/wallet.rs:231]: skip serializing too?

Comment (Thoralf-M) [reply bindings/core/src/method/wallet.rs:231]: Not needed, we never serialize this from Rust

Comment (DaughterOfMars) [reply bindings/core/src/method/wallet.rs:231]: fair enough

**Review by DaughterOfMars — DISMISSED** [review]

**Comment by Alex6323** [comment]
looks good to me, but we could prevent some cloning with a slight refactor if we wanted to at some point.

**Review by Alex6323 — APPROVED** [review]

**Review by DaughterOfMars — APPROVED** [review]

**Pull request merged**